_i18n/de/resources/user-guides/howto_fix_stuck_funds.md

```diff
@@ -1,32 +1,32 @@
-{% include disclaimer.html translated="no" translationOutdated="no" %}
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
 
-Sometimes, your funds will become stuck - you will have some locked funds that never become unlocked. This is how you fix it.
+Manchmal kann es passieren, dass dein Geld feststeckt - du hast gesperrtes Guthaben, das niemals freigegeben wird. So löst du das Problem:
 
-- Load your wallet in monero-wallet-cli.
+- Lade dein Wallet in monero-wallet-cli.
 
-- Type
+- Tippe
 
 > seed
 
-into the command prompt. Write down your 25 word seed, if you haven't already. This is the best way to make sure you don't loose access to your funds.
+in die Befehlszeile ein. Falls noch nicht geschehen, schreibe deinen 25-Wort-Seed nieder. Dies ist der beste Weg um sicherzustellen, dass du nicht den Zugriff auf deine Gelder verlierst.
```

Comment:
```suggestion
in die Befehlszeile ein. Falls noch nicht geschehen, schreibe deinen 25-Wort-Seed nieder. Dies ist der beste Weg um sicherzustellen, dass du den Zugriff auf deine Gelder nicht verlierst.
```

"deine Gelder" doesn't sound right.. "dein Geld", "dein Guthaben", "dein Vermögen", "deine geliebten Moneros"?

Reply:
Although I really like "deine geliebten Monero(s)" I will go with "dein Vermögen". I used "Gelder" before already because I think it fits in some contexts, but it may sound odd here.